src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_test.py

```diff
@@ -1804,6 +1817,348 @@ def test_fuzz_with_mutations_using_ml_rnn(self, mock_execute, *_):
+                  1769,
+              u'initial_feature_coverage':
+                  4958,
+              u'expected_duration':
+                  2650,
+              'fuzzer':
+                  u'libfuzzer_fake_fuzzer',
+              u'fuzzing_time_percent':
+                  0.0,
+              'job':
+                  u'job_name',
+              'kind':
+                  u'TestcaseRun',
+              u'leak_count':
+                  0,
+              u'log_lines_from_engine':
+                  65,
+              u'log_lines_ignored':
+                  8,
+              u'log_lines_unwanted':
+                  0,
+              u'manual_dict_size':
+                  0,
+              u'max_len':
+                  80,
+              u'merge_edge_coverage':
+                  1769,
+              u'new_edges':
+                  0,
+              u'new_features':
+                  0,
+              u'new_units_added':
+                  11,
+              u'new_units_generated':
+                  55,
+              u'number_of_executed_units':
+                  258724,
+              u'oom_count':
+                  0,
+              u'peak_rss_mb':
+                  103,
+              u'recommended_dict_size':
+                  0,
+              u'slow_unit_count':
+                  0,
+              u'slow_units_count':
+                  0,
+              u'slowest_unit_time_sec':
+                  0,
+              u'startup_crash_count':
+                  0,
+              u'strategy_dataflow_tracing':
+                  0,
+              u'strategy_corpus_mutations_radamsa':
+                  0,
+              u'strategy_corpus_mutations_ml_rnn':
+                  0,
+              u'strategy_corpus_subset':
+                  0,
+              u'strategy_fork':
+                  1,
+              u'strategy_mutator_plugin':
+                  0,
+              u'strategy_random_max_len':
+                  0,
+              u'strategy_recommended_dict':
+                  0,
+              u'strategy_value_profile':
+                  0,
+              u'timeout_count':
+                  0,
+              u'timeout_limit':
+                  25,
+              'timestamp':
+                  1337.0
+          })
+
+      # Output printed.
+      self.assertIn(self.no_crash_output, mock_stdout.getvalue())
+      expected_command = (
+          'Command: /fake/build_dir/fake_fuzzer '
+          '-max_len=80 -rss_limit_mb=2048 -timeout=25 '
+          '-collect_data_flow=/fake/dfsan_build/fake_fuzzer -fork=1 '
+          '-artifact_prefix=/fake/ -max_total_time=2650 -print_final_stats=1 '
+          '/fake/inputs-disk/temp-1337/new /fake/corpus_dir')
+      self.assertIn(expected_command, mock_stdout.getvalue())
+      self.assertIn('Bot: test-bot', mock_stdout.getvalue())
+      self.assertIn('Time ran:', mock_stdout.getvalue())
+
+  @mock.patch('bot.fuzzers.libFuzzer.launcher.add_recommended_dictionary',
```

Comment:
I think we should use test_helpers.patch in setUp, instead of repeating the same patch decorators for add_recommended_dictionary, cpu_count and stdout.

Reply:
Not every test in this class patches those, so not sure about moving everything to the `setUp`. Added a note to #594.